Tests/PhishingDetectionTests/PhishingDetectionServiceTests.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+
+import XCTest
+@testable import BrowserServicesKit
+@testable import PhishingDetection
+
+class PhishingDetectionServiceTests: XCTestCase {
+    var service: PhishingDetectionServiceProtocol!
+
+    override func setUp() {
+        super.setUp()
+        let mockClient = MockPhishingDetectionClient()
+        service = PhishingDetectionService(apiClient: mockClient)
+    }
+
+    override func tearDown() {
+        service = nil
+        super.tearDown()
+    }
+
+    func testUpdateFilterSet() async {
+        await service.updateFilterSet()
+        XCTAssertFalse(service.filterSet.isEmpty, "Filter set should not be empty after update.")
+    }
+
+    func testUpdateHashPrefixes() async {
+        await service.updateHashPrefixes()
+        XCTAssertFalse(service.hashPrefixes.isEmpty, "Hash prefixes should not be empty after update.")
+        XCTAssertEqual(service.hashPrefixes, [
+            "aa00bb11",
+            "bb00cc11",
+            "cc00dd11",
+            "dd00ee11",
+            "a379a6f6"
+        ])
+    }
+
+    func testGetMatches() async {
```

Comment:
This API I don’t think is needed and I don’t think it needs not be tested directly